app/helpers/application_helper.rb

```diff
@@ -38,10 +42,23 @@ def ban_error_message(domains, valid_until)
     if valid_until
       t(:banned_completely, valid_until: valid_until.to_date)
     else
-      t(:banned, domain_names: domains.join(', '))
+      active_domains = check_active_auctions_for(domains)
+      active_domains.count.positive? ? t(:banned, domain_names: active_domains.join(', ')) : ''
```

Comment:
Avoid cluttering global namespace (`en.banned`).

Reply:
Moved all the translations from here to 'auctions.*'